actor/src/broker.rs

```diff
@@ -109,6 +109,43 @@ impl Broker {
             .fold(0, |p, (_, n)| p + if n.0.is_closed() { 0 } else { 1 })
     }
 
+    /// Synchronously send message via broker.
+    /// # Panics
+    /// Panics if called from within an async runtime.
```

Comment:
This isn't accurate. It panics if called as an asynchronous function, but it can be called sunchronously from within the run-time.

Reply:
Oh yea. That comment is old. Before I was creating a new runtime on the current thread to run async code and that would cause a panic if called from within an existing runtime. Will change the comment.